Sources/StreamChat/APIClient/Endpoints/ModerationEndpoints.swift

```diff
@@ -16,6 +16,44 @@ extension Endpoint {
     }
 }
 
+// MARK: - User blocking
+
+extension Endpoint {
+    static func blockUser(_ userId: UserId, keepChannelsVisible: Bool = false) -> Endpoint<BlockedUsersResponse> {
+        let body: [String: AnyEncodable] = [
+            "blocked_user_id": AnyEncodable(userId),
+            "keep_channels_visible": AnyEncodable(keepChannelsVisible)
+        ]
+        
+        return .init(
+            path: .blockUser,
+            method: .post,
+            queryItems: nil,
+            requiresConnectionId: false,
+            body: body
+        )
+    }
+
+    static func unblockUser(_ userId: UserId) -> Endpoint<EmptyResponse> {
+        .init(
+            path: .unblockUser,
+            method: .post,
+            queryItems: nil,
+            requiresConnectionId: false,
+            body: ["blocked_user_id": userId]
+        )
+    }
+    
+    static func loadBlockedUsers() -> Endpoint<BlockedUsersResponse> {
+        .init(
+            path: .blockUser,
+            method: .get,
+            queryItems: nil,
+            requiresConnectionId: false
+        )
+    }
+}
```

Comment:
Do we need something like this? If so, what should it look like?
We don't have `loadMutedUsers()` nor `loadFlaggedUsers()`.

Reply:
This should be a new action in `CurrentUserController`, something like `getBlockedUsers()`.

Reply:
Created:
```swift
    /// Get all blocked users.
    ///
    /// - Parameter completion: Called when the API call is finished. Called with `Error` if the remote update fails.
    ///
    func getBlockedUsers(completion: ((Error?) -> Void)? = nil) {
        currentUserUpdater.apiClient.request(endpoint: .loadBlockedUsers()) {
            completion?($0.error)
        }
    }
```